examples/twilio-sms-connector/src/main/resources/application.example.properties

```diff
@@ -0,0 +1,35 @@
+########################################################################
```

Comment:
File should be named `application.properties`. Naming it `application.example.properties` would need users to set active profile to be `example`.

Reply:
@asifali-eai spring apps expect `application.properties` to be present when running in default profile. So the way to configure it is like how we did it in twitter connector:
https://github.com/forcedotcom/einstein-bot-channel-connector/blob/dabfbe210bace12680ab1a4249f54d493b2535a5/examples/twitter-connector/src/main/resources/application.properties#L47

You checkin `application.properties` that uses env vars to inject sensitive information as well as custom config. You can have a separate `application-development.properties` that holds your specific keys and make sure that you dont check that in (add it to `.gitignore` so there is no accidental chance of checking it in).

Reply:
Thank you. Renamed `application.example.properties` to `application.properties` and moved out sensitive property values to environment variables.